SwiftReflector/SwiftInterfaceReflector/SwiftInterfaceReflector.cs

```diff
@@ -973,16 +985,23 @@ void AssignSwiftInterfaceFormat (string formatVersion)
 		void AssignSwiftCompilerVersion (string compilerVersion)
 		{
 			// when we get here, we should see something like:
-			// [white-space]*Apple Swift version VERSION (swiftlang-VERSION clang-VERSION)
+			// [white-space]*Apple? Swift version VERSION (swiftlang-VERSION clang-VERSION)
 			var parts = compilerVersion.Trim ().Split (' ', '\t'); // don't know if tab is a thing
 									       // expect in the array:
 									       // 0: Apple
 									       // 1: Swift
 									       // 2: verion
 									       // 3: VERSION
-			if (parts [0] != "Apple" || parts [1] != "Swift" || parts [2] != "version")
+
+			var swiftIndex = Array.IndexOf (parts, "Swift");
+			if (swiftIndex < 0)
+				throw new ArgumentOutOfRangeException (nameof (compilerVersion), $"Expected 'Swift' in the version string, but got {compilerVersion}");
+			if (parts [swiftIndex + 1] != "version")
 				throw new ArgumentOutOfRangeException (nameof (compilerVersion), $"Expected a compiler version string but got {compilerVersion}");
-			if (!Version.TryParse (parts [3], out this.compilerVersion))
+			var version = parts [swiftIndex + 2];
+			if (version.EndsWith ("-dev"))
```

Comment:
EndsWith is culture-aware by default, and it doesn't look like that's needed:

```suggestion
			if (version.EndsWith ("-dev", StringComparison.Ordinal))
```

Reply:
Thanks for getting that. I can't say that I like that nonsense.